FirebaseAppDistribution/Sources/FIRAppDistributionAppDelegateInterceptor.m

```diff
@@ -90,6 +98,11 @@ - (void)appDistributionRegistrationFlow:(NSURL *)URL
   }
 }
 
+- (NSString *)encodedAppId {
```

Comment:
Should we make this a static method because it is stateless? I'm not sure what the style guide dictates about when this is or isn't a good idea, but my java rat brain tends toward make this a static utility if it does not depend on state.

Reply:
Good call, definitely!